Sources/TestSupport/misc.swift

```diff
@@ -72,11 +72,12 @@ public func fixture(name: String, tags: [String] = [], file: StaticString = #fil
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "init"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "user.email", "example@example.com"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "user.name", "Example Example"])
+                try systemQuietly([Git.tool, "-C", dstDir.asString, "config", "commit.gpgsign", "false"])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "add", "."])
                 try systemQuietly([Git.tool, "-C", dstDir.asString, "commit", "-m", "msg"])
```

Comment:
I'm happy to fix that here, but in what capacity do you mean? Are there `GitRepository` functions somewhere that I'm missing for tagging/adding/committing?